lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -202,43 +199,39 @@ def make_vetext_request(request_body):
         "body": request_body.get("Body", "")
     }
 
-    json_data = json.dumps(body)
-
-    logger.info(f"Making POST Request to VeText using: ${domain}${path}")
-    logger.debug(f"json dumps: {json_data}")
+    endpoint_uri = f"https://{domain}{path}"
 
-    connection = None
+    logger.info(f"Making POST Request to VeText using: {endpoint_uri}")
+    logger.debug(f"json dumps: {json.dumps(body)}")
 
     try:
-        connection = http.client.HTTPSConnection(domain, timeout=6, context=ssl._create_unverified_context())
-        logger.info("generated connection to VeText")
-
-        connection.request(
-            'POST',
-            path,
-            json_data,
-            headers)
-
-        response = connection.getresponse()
-
-        logger.info(f"VeText call complete with response: {response.status}")
-        logger.debug(f"VeText response: {response}")
-
-        if response.status == 200:
-            return response
-
-        logger.error("VeText call failed.")
-    except http.client.HTTPException as e:
-        logger.error("HttpException With Call To VeText")
+        # setting verify to false at the direction of VeText
+        response = requests.post(
+            endpoint_uri,
+            verify=False,
+            json=body,
+            timeout=HTTPTIMEOUT,
+            headers=headers
+        )
+        response.raise_for_status()
+        
+        response_content = response.content
```

Comment:
```suggestion
        if response.ok:
            response_json = response.json()
```

Reply:
`response.content` generates difficult to read strings. I cannot see why we'd use that over `response.json()` for this use case.